css/index.css

```diff
@@ -5,17 +5,16 @@ body{
 .blocks{
     background-color: white;
     height: auto;
-    min-height: 200px;
-    height: 350px;
+    min-height: 350px;
```

Comment:
I set the height to auto and defined a min-height of 350 pixels. This sets all the boxes to be the same height at first but allows the 'guesses' box to grow if the user needs more than 8 guesses. Previously it was overflowing the container.